client/state/happychat/selectors/get-happychat-auth.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
```

Comment:
Could we find a better place for this, does not look like a state selector.

Reply:
I'm a bit at lost about how this could be improved. Do you have any suggestion?

Reply:
I'm not sure how I feel about having a selector that fetches data other than from the global state tree either. If this is a blocker perhaps I could move this function to `state/happychat/utils.js`?

Reply:
This is kinda tricky but I guess a helper or utils sepearate file is better than a selector.